Makefile

```diff
@@ -1,5 +1,16 @@
 build:
 	CGO_ENABLED=0 GO111MODULE=on go build -mod vendor -o _output/bin/cloud-network-config-controller cmd/cloud-network-config-controller/main.go
 test:
-	go test ./... -count=1 -race
+	# This is commenting out the racy tests. This is because the racy tests are
```

Comment:
I don't understand this. You aren't commenting out any tests, you're just telling go to not warn you that your tests have race conditions in them. But regardless of whether you let go warn you about it or not, the racy tests will still cause unit test flakes...

(Also, typo in the commit message: "Disbale")

Reply:
The test file containing the racy tests have this defined: 
https://github.com/openshift/cloud-network-config-controller/blob/df7a65440de18c14730605e3a1662399174b28dc/pkg/controller/cloudprivateipconfig/cloudprivateipconfig_controller_racy_test.go#L1
So it will only run for `go test -race`. Hence by commenting that part out from the Makefile, I am effectively "commenting out the test"

Noted for the spelling mistake, will update now.

Reply:
I could maybe re-phrase this to make it clearer.